zkevm-circuits/src/witness/taiko.rs

```diff
@@ -33,4 +33,11 @@ pub struct Taiko {
     pub max_transactions_per_block: u64,
     /// maxBytesPerTxList
     pub max_bytes_per_tx_list: u64,
+
+    /// anchor gas cost
+    pub anchor_gas_cost: u64,
+    /// anchor from
+    pub anchor_from: Address,
```

Comment:
I think `from` and `to` aren't needed.
`from` is our golden touch address
`to` is the `l2_contract` address

Reply:
`from` can be static yes. I think the idea with `to` is that we can simply change the address in the smart contract when it changes and we don't have to update the circuits.